internal/pkg/deploy/cloudformation/stack/testdata/workloads/svc-foo.stack.yml

```diff
@@ -345,16 +345,9 @@ Resources:
         AwsvpcConfiguration:
           AssignPublicIp: ENABLED
           Subnets:
-            - Fn::Select:
```

Comment:
Does this break anywhere else in the template? I seem to recall us using `PublicSubnet1` and `PublicSubnet2` or the like in some spots in the CF.

Reply:
Good spidey-sense. I think we've got it covered-- I believe any assumptions of those are with Copilot-generated VPCs and will have the appropriate subnets.